README.md

```diff
@@ -602,6 +608,10 @@ A curated list of awesome C++ (or C) frameworks, libraries, resources, and shiny
 * [protobuf-c](https://github.com/protobuf-c/protobuf-c) - Protocol Buffers implementation in C. [BSD]
 * [SimpleBinaryEncoding](https://github.com/real-logic/simple-binary-encoding) - encoding and decoding application messages in binary format for low-latency applications. [Apache2]
 
+## Sorting
```

Comment:
Please add a link to the new category in the index section.

Reply:
Done